core/drivers/crypto/caam/utils/utils_mem.c

```diff
@@ -203,13 +77,13 @@ static void *mem_alloc(size_t size, uint8_t type)
 
```

Comment:
Perhaps "(normal)" is no more meaningful? (same at other places below)

Reply:
I will remove it